src/cosmos/info/thorchainInfo.ts

```diff
@@ -1,14 +1,48 @@
+import { fromBech32 } from '@cosmjs/encoding'
+import { EncodeObject, Registry } from '@cosmjs/proto-signing'
+import { coin } from '@cosmjs/stargate'
```

Comment:
It is *ABSOLUTELY FORBIDDEN* to import SDK stuff from the info files. The info files can only contain plain-old JSON-style data.

To make this work, you will have to do `switch (pluginId) { case 'thorchain': ... }` in the Cosmos tools, and then put all the per-chain stuff in there. The tools run "on the other side of the fence" where talking to the SDK is allowed.